config/locales/en.yml

```diff
@@ -380,6 +380,7 @@ en:
           no_description: "No description"
         step_table:
           sidebar_name: "Table" # TODO
```

Comment:
This line `sidebar_name` is probably no longer needed, at least I couldn't `grep` anything.